recipes-kernel/linux/linux-raspberrypi_6.6.bbappend

```diff
@@ -32,3 +32,16 @@ SRC_URI:append = " \
     file://pcie1-passthrough.dtso;subdir=git/arch/${ARCH}/boot/dts/broadcom \
     file://0001-drivers-mmc-host-sdhci-brcmstb-fix-no-pinctrl-case.patch \
 "
+
+SRC_URI:append = "${@bb.utils.contains('MACHINE_FEATURES', 'scmi', 'file://scmi-config.cfg', '', d)}"
+
+SRC_URI:append = " \
+    file://0001-firmware-arm_scmi-Increase-the-maximum-opp-count-in-.patch \
```

Comment:
You may combine this SRC_URI:append with the previous one. So, final construction will look:

```
SRC_URI:append = " \
    ${@bb.utils.contains('MACHINE_FEATURES', 'scmi', 'file://scmi-config.cfg', '', d)} \
    file://0001-firmware-arm_scmi-Increase-the-maximum-opp-count-in-.patch \
...
"

Reply:
I specifically split it so it can be seen what is added by which feature. Then more easy to update in the future - during migration for example.